python-sdk/src/astro/databases/sqlite.py

```diff
@@ -71,7 +71,7 @@ def create_schema_if_needed(self, schema: str | None) -> None:
         Since SQLite does not have schemas, we do not need to set a schema here.
         """
 
-    def schema_exists(self, schema: str) -> bool:
+    def schema_exists(self, schema: str) -> bool:  # skipcq PYL-W0613,PYL-R0201
```

Comment:
@pankajastro Static method?

Reply:
two issue 
- override of base class method 
- schema is unused param